code/modules/surgery/organs/organ_internal.dm

```diff
@@ -0,0 +1,314 @@
+#define PROCESS_ACCURACY 10
+
+/obj/item/organ/internal
+	origin_tech = "biotech=2"
+	force = 1
+	w_class = 2
+	throwforce = 0
+	var/zone = "chest"
+	var/slot
+	vital = 0
+	var/organ_action_name = null
+
+/obj/item/organ/internal/proc/insert(mob/living/carbon/M, special = 0)
+	if(!iscarbon(M) || owner == M)
+		return
+
+	var/obj/item/organ/internal/replaced = M.get_organ_slot(slot)
+	if(replaced)
+		replaced.remove(M, special = 1)
+
+	owner = M
+	M.internal_organs |= src
+	loc = null
```

Comment:
This line here means that due to organ.dm: process(), these organs will both: 
* lack an owner
* Rot from inside the host's body.

You probably want to set their loc to be in the mob itself, so you don't have that happen.